### ISSUE TITLE
register multiple services, which are used?

### ISSUE THREAD

**Issue by neuecc** [hadashiA/VContainer#189]
```csharp
builder.Register<IFoo, Foo>();
builder.Register<IFoo, Bar>();

builder.Build().Resolver<IFoo>(); // which type?
```

Is this behavior undefined? First or Last?

`Microsoft.Extensions.DependencyInjection` uses `Last` registered.
This is useful for overriding arbitrarily after a plugin has registered.

**Comment by hadashiA** [comment]
Currently, I think the First is always returned. 

>  This is useful for overriding arbitrarily after a plugin has registered.

I see. That looks better. 
I'm going to fix it so that Last is returned.

**Comment by hadashiA** [comment]
Released in [v1.7.2](https://github.com/hadashiA/VContainer/releases/tag/1.7.2) .
Thanks for the advice .
